purchases/src/main/kotlin/com/revenuecat/purchases/Factories.kt

```diff
@@ -38,14 +38,29 @@ internal fun JSONObject.buildPurchaserInfo(): PurchaserInfo {
             }
         } else null
 
+    val firstSeen =
```

Comment:
It can be null if it has been cached before. But I don't want to make it nullable. Should I default it to `Date.now()`? @tonycosentini

Reply:
How can it be null if it was already cached?  

Isn't `now()` also a bit dangerous since it can drastically differ from what will eventually come down?

Reply:
Sorry I meant: "It can be null if it has NOT been cached before"

I think I will just make it nullable since it's safer

Reply:
Yeah, I think that's more clear.